wiki/Tournaments/OIT/2021/en.md

```diff
@@ -0,0 +1,142 @@
+- Hidden
+  - [Rin - Moriya set 08 ReEdit \~ Youkai no Yama (jonathanlfj) \[Extra Stage\]](https://osu.ppy.sh/beatmapsets/751866#osu/1582806)
+  - [Emily Osment - Hero in Me (AJT) \[Save The Day\]](https://osu.ppy.sh/beatmapsets/1229079#osu/2555552)
+- HardRock
+  - [UNDEAD CORPORATION - Adore Your Pain (jomnhnk) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/76745#osu/217083)
+  - [Kairiki bear feat. Hatsune Miku - Telesto Teles (Ryuusei Aika) \[wafer's Expert\]](https://osu.ppy.sh/beatmapsets/1196329#osu/2500705)
+- DoubleTime
+  - [Hanatan - Kokoro (sodarose) \[Miracle\]](https://osu.ppy.sh/beatmapsets/1005463#osu/2104338)
+  - [Sunpon - Skyscraper Grief (Natteke) \[Happy's Lunatic\]](https://osu.ppy.sh/beatmapsets/15200#osu/69390)
+
+## Ruleset
+
+### General rules
+
+1. Beatmap scoring is based on **[ScoreV2](/wiki/Score#scorev2).**
+2. The mapsets for each round will be announced by the Tournament Management in advance before the actual matches take place.
+3. Match schedules will be predetermined by the Tournament Management. If there are any player(s) who are unable to attend the current schedule for any reason, all other affected players may apply and settle for a reschedule at the #scheduling channel in the tournament's Discord server.
+4. A referee will create a multiplayer room 10 minutes in advance and will start to send out invites.
+5. If a player does not show up within **10 minutes** of the start time, their opponent wins by default.
+6. If no staff or referee is available, the match will be postponed.
+7. **NoFail will be enforced in all beatmaps.** This is to ensure that the points are to be awarded more fairly towards players who perform better in general during the course of the beatmap regardless of their remaining health at the end.
+8. If a player disconnects, it will be treated as if they failed the beatmap.
+   - Disconnects that occur within a few seconds after the beatmap is started by the referee can be rematched.
+9. If a player disconnects between beatmaps, the match can be delayed up to 15 minutes at most.
+   - In case the disconnected player failed to report back after the maximum allowance of 15 minutes had passed, the opposing player may be declared to be winning the match by default.
+10. Lag is not a valid reason to nullify a beatmap.
+11. If there are any problems during the match occurence, the Tournament Management will make a decision based on referee's report.
+12. It is expected that all players be polite and respectful to each other. Penalties will be given if players violate.
+    - If a player is found to be engaging in an act of offensive provocation, the said player responsible for the provocation may be either disqualified right away from OIT 2021 and/or blacklisted from the future installment(s) of OIT.
+    - Usage of any illegal programs is strictly prohibited and will be straight up reported to the osu!staff at will.
```

Comment:
- to me, "illegal" is something that is forbidden by the law. should probably reword this to underline that it breaks osu!'s rules instead
- `osu!staff` -> `osu!team`